cli/report/__init__.py

```diff
@@ -46,22 +46,20 @@
 @click.option(
     "--gcs_bucket",
     help="The name of the GCS bucket that holds OpenShift CI logs",
-    required=True,
     default="origin-ci-test",
     type=click.STRING,
 )
 @click.option(
     "--firewatch_config_path",
     help="The path to the firewatch configuration file",
     required=False,
-    type=click.STRING,
+    type=click.Path(),
 )
 @click.option(
     "--jira_config_path",
     help="The path to the jira configuration file",
-    required=True,
     default="/tmp/jira.config",
-    type=click.STRING,
+    type=click.Path(),
```

Comment:
use `exists=True` when you expect the file to be there, and let click handle the fail if it's not.